src/components/TopBar/TopBar.js

```diff
@@ -3,22 +3,35 @@ import AppBar from '@mui/material/AppBar';
 import Toolbar from '@mui/material/Toolbar';
 import Typography from '@mui/material/Typography';
 import LoginToggleButton from "./LoginToggleButton";
-import {Link} from "react-router-dom";
+import { Link } from "react-router-dom";
 import "./topbar.css";
-import {CssBaseline} from "@mui/material";
+import { CssBaseline } from "@mui/material";
+import Box from '@mui/material/Box';
+import CardMedia from '@mui/material/CardMedia';
 
 const TopBar = () => {
   return (
+    <div>
       <CssBaseline>
         <AppBar position="static">
           <Toolbar>
-            <Typography variant="h6" component="div" sx={{flexGrow: 1}}>
-              <Link className="top-bar-link" to="/">Rave</Link>
+            <Typography variant="h6" component="div" sx={{ flexGrow: 1 }}>
+              <Link className="top-bar-link" to="/">
+                <Box sx={{ position: 'relative', width: 70, height: '100%', display: 'flex', alignItems: 'center' }}>
```

Comment:
We want the link to be just for the media box (for redirection to home), not the entire top nav bar.